README_sdl.txt

```diff
@@ -50,11 +50,12 @@ cd build_emscripten
 with each binary, so we make sure each example and demo has such a
 folder:
 
-mkdir -p demos/speed/data
```

Comment:
This needs to be reverted as this was actually correct: the speed demo has no data directory, so it won't build without this.

For examples, instead of this change, could you add `make copy_example_data` before `make ex_draw_bitmap` below? I'll look into fixing the cmake dependency later so this happens automatically.

Reply:
I can tell that this timer is not functioning correctly (I added a print statement in the loop and `al_get_timer_count(inc_counter)` is alway zero): https://github.com/connorjclark/allegro5/blob/f41f49d65c89fdb214b4dd49172b5adfbf16c4ec/demos/speed/main.c#L64-L64

could it be because it isn't registered to an event queue?

Reply:
That demo is written very unidiomatically, as it's a pretty basic sourceport of the original written for Allegro 4. The issue is that it wasn't calling a "heartbeat" function everywhere. I added a small hack to get it half working, but it really needs a rewrite: https://github.com/liballeg/allegro5/commit/42fe8739a6dd286e84052ba927103f4edcdbb22b